Source/SwiftLintFramework/Rules/Lint/UnusedImportRule.swift

```diff
@@ -28,6 +28,12 @@ public struct UnusedImportRule: CorrectableRule, ConfigurationProviderRule, Anal
             """
             import UnknownModule
             func foo(error: Swift.Error) {}
+            """,
+            """
+            import Foundation
+            import ObjectiveC
+            let 👨‍👩‍👧‍👦 = #selector(NSArray.contains(_:))
+            👨‍👩‍👧‍👦 == 👨‍👩‍👧‍👦
```

Comment:
`ObjectiveC` module introduces an overload of the `==` operator:
```swift
public static func == (lhs: Selector, rhs: Selector) -> Bool
```
and even though `Selector` is introduced by the `ObjectiveC` module it can be indirectly created with `#selector`. This operator overload is the only way I found to reproduce the original issue in the unit tests.